### ISSUE TITLE
Status/Navigation bar being invisible

### ISSUE THREAD

**Issue by carstenhag** [ccrama/Slide-iOS#1176]
The status bar (the thing at the very top of the screen) is blank on the main screen and the post screen. Possibly also in others.

It's only showing the text as expected on the sliding profile page.

![image](https://user-images.githubusercontent.com/3776997/111024920-793f3d80-83e1-11eb-8ada-367d7547736e.png)
![image](https://user-images.githubusercontent.com/3776997/111024923-7e03f180-83e1-11eb-9801-9dfa6c7bbe42.png)
![image](https://user-images.githubusercontent.com/3776997/111024927-83613c00-83e1-11eb-9030-03188f7b8fe0.png)

**Comment by ccrama** [comment]
Thank you for the report! Do you have "hide status bar on scroll" enabled in Settings > General?

**Comment by carstenhag** [comment]
No, that setting was turned off. Also doesn't work when turning it on.
